tests/functional/aws-node-sdk/test/multipleBackend/get.js

```diff
@@ -5,6 +5,7 @@ import BucketUtility from '../../lib/utility/bucket-util';
 const bucket = 'buckettestmultiplebackendget';
 const memObject = 'memobject';
 const fileObject = 'fileobject';
+const awsObject = 'awsobject';
 const emptyObject = 'emptyObject';
 const body = Buffer.from('I am a body', 'utf8');
```

Comment:
I would be reluctant to run such test in the CI (because of time to send all bytes to AWS and $$). Although I agree this must be tested, at least once, and may be in Integration instead or could be run as part of a non-reg test suite before a release?